apps/crm/crm-e2e/src/fixtures/create-org.ts

```diff
@@ -0,0 +1,114 @@
+});
+
+const permissions = createPermissions({
+  id: blockframesOrgAdminId,
+  roles: { [blockframesAdminId]: 'superAdmin' },
+});
+
+export const admin = {
+  user: blockframesAdmin,
+  org: blockframesAdminOrg,
+  permissions,
+};
+
+//* new organizations
+
+export type Newcomer = {
+  status: OrganizationStatus;
+  apps: Exclude<App, 'crm' | 'financiers'>[];
+  modules: Module[];
+  data: {
+    org: Organization;
+    user: User;
+  };
+};
+
+export const newcomers: Newcomer[] = [
+  { status: 'accepted', apps: ['catalog'], modules: ['dashboard'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['catalog'], modules: ['marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['catalog'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['catalog', 'festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['catalog', 'festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'accepted', apps: ['catalog', 'festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'pending', apps: ['catalog'], modules: ['dashboard'], data: createOrgWithuser() },
+  { status: 'pending', apps: ['festival'], modules: ['marketplace'], data: createOrgWithuser() },
+  { status: 'pending', apps: ['catalog', 'festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+  { status: 'onhold', apps: ['catalog'], modules: ['dashboard'], data: createOrgWithuser() },
+  { status: 'onhold', apps: ['festival'], modules: ['marketplace'], data: createOrgWithuser() },
+  { status: 'onhold', apps: ['catalog', 'festival'], modules: ['dashboard', 'marketplace'], data: createOrgWithuser() },
+];
+
+//* functions
+
+function createOrgWithuser() {
+  const data = fakeUserData();
+  return {
+    org: createOrganization({
+      name: data.company.name,
+      email: 'contact@' + data.email.split('@')[1],
+      activity: randomValue(orgActivity),
+      addresses: {
+        main: {
+          country: randomValue(territories),
+          region: '', // not in the form
+          city: faker.address.cityName(),
+          zipCode: faker.address.zipCode(),
+          street: 'E2E street',
+          phoneNumber: faker.phone.phoneNumber(),
+        },
+      },
+    }),
+    user: createUser({
+      firstName: data.firstName,
+      lastName: data.lastName,
+      email: data.email,
+    }),
+  };
+}
+
+function randomValue(obj) {
+  const keys = Object.keys(obj);
+  return obj[keys[(keys.length * Math.random()) << 0]];
+}
```

Comment:
```suggestion
function getRandom<S extends Scope>(base: S) {
  const keys = Object.keys(staticModel[base]);
  return keys[(keys.length * Math.random()) << 0];
}
```

Like this you can do `getRandom('orgActivity')` or `getRandom('territories')
`